spec.md

```diff
@@ -1428,6 +1434,109 @@ For success responses, the following fields are defined:
 }
 ```
 
+## Fetching a Service Instance
+
+If `"instances_retrievable" :true` is declared for a service in the
+[Catalog](#catalog-management) endpoint, Service Brokers MUST support this
+endpoint for all plans of the service.
+
+### Request
+
+##### Route
+`GET /v2/service_instances/:instance_id`
+
+`:instance_id` is the identifier of a previously provisioned instance.
+
+##### cURL
+```
+$ curl 'http://username:password@broker-url/v2/service_instances/:instance_id' -X GET -H "X-Broker-API-Version: 2.13"
+```
+
+### Response
+
+| Status Code | Description |
+| --- | --- |
+| 200 OK | The expected response body is below. |
+| 404 Not Found | MUST be returned if the Service Instance does not exist or if a provisioning operation is still in progress. |
```

Comment:
In other ops we return a 410 not 404 when the Instance doesn't exist. Should we be consistent?

Reply:
during 4/11 F2F we agreed to keeping it a 404